<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmp</class>
 <widget class="QDialog" name="cmp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>请输入</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>82</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>职工1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>82</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>职工2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>四月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>五月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>六月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>七月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>八月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>九月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>十月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>十一月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>十二月</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>对比月份</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
